statistics/handle/bootstrap.go

```diff
@@ -32,7 +32,15 @@ import (
 	"go.uber.org/zap"
 )
 
-func (h *Handle) initStatsMeta4Chunk(is infoschema.InfoSchema, cache *statsCache, iter *chunk.Iterator4Chunk) {
+const (
+	// defaultTopNnum ,defaultBucketsSize are defined in planner/core/planbuilder.go analyzeOptionDefault.
+	defaultTopNnum     = 20
```

Comment:
Can you use OptionDefault directly?

Reply:
OptionDefault are in https://github.com/pingcap/tidb/blob/master/planner/core/planbuilder.go#L1632-#L1637 ，which are not easy to get directly...